# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../parted/parted-3.0-glibc-2.16-no-gets.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Since the 2.16 release of the glibc, 'gets' is not defined in the gnulib anymore

This is a hack until upstream pulls in a new gnulib version

Within gnulib this issue was already adsressed in the following commit:

http://git.savannah.gnu.org/gitweb/?p=gnulib.git;a=commitdiff;h=66712c23388e93e5c518ebc8515140fa0c80734

--- parted-3.0/lib/stdio.in.h.orig	2012-08-05 15:10:07.521753187 +0200
+++ parted-3.0/lib/stdio.in.h	2012-08-05 15:11:12.713387639 +0200
@@ -735,7 +735,6 @@
 /* It is very rare that the developer ever has full control of stdin,
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
 
